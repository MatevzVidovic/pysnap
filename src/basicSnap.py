




def test_snap_outer(snapshot):
    from basicExample import outer
    snapshot.assert_match(outer("ena", "dva"))