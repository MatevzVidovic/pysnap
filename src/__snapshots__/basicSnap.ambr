# serializer version: 1
# name: test_snap_outer
  'enadvaenadva'
# ---
